README.md

```diff
@@ -101,6 +101,29 @@ Send a pull request to any of our open source repositories on Github. Check our
 ## Contributing
 The project can only accept contributions which are licensed under the [Apache License 2.0](LICENSE). For further information please see our [Contribution Guidelines](CONTRIBUTING.md).
 
+## Changelog
+
+Install the generator
+```
+go get -u github.com/git-chglog/git-chglog/cmd/git-chglog
+```
+
+Usage
+```
+git-chglog -o CHANGELOG.md
```

Comment:
`git-chglog --next-tag vx.y.z -o CHANGELOG.md`

Is better, because  at the time of generation the tag won't be there yet, it will be created with the release commit.